includes/abilities-api/class-wp-ability-category.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Abilities API
+ *
+ * Defines WP_Ability_Category class.
+ *
+ * @package WordPress
+ * @subpackage Abilities API
+ * @since n.e.x.t
+ */
+
+declare( strict_types = 1 );
+
+/**
+ * Encapsulates the properties and methods related to a specific ability category.
+ *
+ * @since n.e.x.t
+ *
+ * @see WP_Abilities_Category_Registry
+ */
+class WP_Ability_Category {
```

Comment:
Any reason why we need to support extending this for WP6.9? Making the class final and/or the props private gives us more time to iterate and figure out what sort of categorization Abilities actually need.

```suggestion
final class WP_Ability_Category {
```

Reply:
I agree with this.
@gziolo @JasonTheAdams ?

Reply:
If we don't want it to be extended, then `final` makes sense. It's a familiar pattern there.

Reply:
Done